client/components/thank-you/index.tsx

```diff
@@ -34,12 +34,27 @@ const ThankYouContainer = styled.div< ThankYouThemeProps >`
 	}
 `;
 
-const ThankYouHeader = styled.div`
+const ThankYouHeader = styled.div< ThankYouThemeProps >`
 	width: 100%;
-	height: 240px;
-	background-color: var( --studio-gray-0 );
 	display: flex;
 	justify-content: center;
+	background-color: ${ ( props ) => props.theme.colors.backgroundColorHeader };
```

Comment:
We need to update the Marketplace theme to include these new theme values/properties
_OR_
we should be using one central theme to drive the basic behaviour. And if we're doing that, then it may make sense to just push these values into CSS!